var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -0,0 +1,410 @@
+                '-DSEACASProj_ENABLE_SEACASSVDI:BOOL=%s' % (
+                    'ON' if '+svdi' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSlice:BOOL=%s' % (
+                    'ON' if '+slice' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSupes:BOOL=%s' % (
+                    'ON' if '+supes' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSuplib:BOOL=%s' % (
+                    'ON' if '+suplib' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSuplibC:BOOL=%s' % (
+                    'ON' if '+suplib' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSuplibCpp:BOOL=%s' % (
+                    'ON' if '+suplib' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASTxtexo:BOOL=%s' % (
+                    'ON' if '+txtexo' in spec else 'OFF'),
+            ])
+
+        if '+kokkos' in spec:
+            if '+openmp' in spec:
+                options.extend([
+                    '-DKokkos_ENABLE_OpenMP:BOOL=ON'
+                ])
+
+            if '+cuda' in spec:
+                options.extend([
+                    '-DSEACASProj_ENABLE_Kokkos:BOOL=ON',
+                    '-DTPL_ENABLE_CUDA:Bool=ON',
+                    '-DCUDA_TOOLKIT_ROOT_DIR:PATH=${CUDA_PATH}',
+                    '-DKOKKOS_ENABLE_DEPRECATED_CODE:BOOL=OFF',
+                    '-DKokkos_ENABLE_Pthread:BOOL=OFF',
+                ])
+            else:
+                options.extend([
+                    '-DSEACASProj_ENABLE_Kokkos:BOOL=ON',
+                    '-DSEACASProj_ENABLE_OpenMP:Bool=ON',
+                    '-DKOKKOS_ENABLE_DEPRECATED_CODE:BOOL=OFF',
+                    '-DKokkos_ENABLE_Pthread:BOOL=OFF',
+                ])
+        else:
+            options.extend([
+                '-DSEACASProj_ENABLE_Kokkos:BOOL=OFF'
+            ])
+
+        if '+thread_safe' in spec:
+            options.extend([
+                '-DSEACASExodus_ENABLE_THREADSAFE:BOOL=ON',
+                '-DSEACASIoss_ENABLE_THREADSAFE:BOOL=ON',
+            ])
+
+        # ######################### TPLs #############################
+
+        # Note: -DXYZ_LIBRARY_NAMES= needs semicolon separated list of names
+        options.extend([
+            '-DTPL_ENABLE_Netcdf:BOOL=ON',
+            '-DNetCDF_ROOT:PATH=%s' % spec['netcdf'].prefix,
+            '-DTPL_ENABLE_X11:BOOL=%s' % (
+                'ON' if '+x11' in spec else 'OFF'),
+        ])
+
+        if '+hdf5' in spec:
+            options.extend([
+                '-DTPL_ENABLE_HDF5:BOOL=ON',
+                '-DHDF5_INCLUDE_DIRS:PATH=%s' % spec['hdf5'].prefix.include,
+                '-DHDF5_LIBRARY_DIRS:PATH=%s' % spec['hdf5'].prefix.lib
+            ])
+        else:
+            options.extend(['-DTPL_ENABLE_HDF5:BOOL=OFF'])
+
+        if '+metis' in spec:
+            options.extend([
+                '-DTPL_ENABLE_METIS:BOOL=ON',
+                '-DMETIS_LIBRARY_DIRS=%s' % spec['metis'].prefix.lib,
```

Comment:
I think you're missing a metis dependency up above.

Reply:
done.